flink-runtime/src/test/java/org/apache/flink/runtime/executiongraph/IntermediateResultPartitionTest.java

```diff
@@ -51,7 +51,8 @@ public void testPipelinedPartitionConsumable() throws Exception {
 		assertFalse(partition2.isConsumable());
 
 		// Not consumable if failover happens
-		result.resetForNewExecution();
```

Comment:
Why not keeping this logic? An `IntermediateResult` can be reset which resets all its `IntermediateResultPartitions`.

Reply:
Sure we can keep it for test purposes.